src/sentry/notifications/platform/slack/provider.py

```diff
@@ -13,18 +24,37 @@
 )
 from sentry.organizations.services.organization.model import RpcOrganizationSummary
 
-# TODO(ecosystem): Figure out a way to use 'SlackBlock' type
-type SlackRenderable = Any
+
+class SlackRenderable(TypedDict):
+    blocks: list[Block]
 
 
 class SlackRenderer(NotificationRenderer[SlackRenderable]):
-    provider_key = NotificationProviderKey.DISCORD
+    provider_key = NotificationProviderKey.SLACK
 
     @classmethod
     def render[DataT: NotificationData](
         cls, *, data: DataT, rendered_template: NotificationRenderedTemplate
     ) -> SlackRenderable:
-        return {}
+        subject = HeaderBlock(text=PlainTextObject(text=rendered_template.subject))
+        body = SectionBlock(text=MarkdownTextObject(text=rendered_template.body))
+
+        actions_block = ActionsBlock(elements=[])
+        for action in rendered_template.actions:
+            actions_block.elements.append(
+                ButtonElement(text=action.get("label", ""), url=action.get("link", ""))
+            )
+
+        blocks = [subject, body, actions_block]
+
+        if rendered_template.footer:
+            footer = SectionBlock(text=MarkdownTextObject(text=rendered_template.footer))
+            blocks.append(footer)
+        if rendered_template.chart:
```

Comment:
🤔 should `chart` just be part of the `body` rendering? (i'm thinking of this as like HTML; where the `chart` would just be a sub-element of `body`)